mirte_telemetrix/scripts/ROS_telemetrix_aio_api.py

```diff
@@ -383,12 +383,20 @@ def __init__(self, board, servo_obj):
         self.board = board
         self.pins = get_pin_numbers(servo_obj)
         self.name = servo_obj["name"]
+        if "min_pulse" in servo_obj:
+            self.min_pulse = servo_obj["min_pulse"]
+        else:
```

Comment:
I think I prefer first setting self.min_pulse = 544. And then only change this when "min_pulse" is set. So removing the else.

Reply:
Changed, that is indeed better.